src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddAndApproveDraftApprenticeshipsCommandHandler.cs

```diff
@@ -35,6 +35,7 @@ public async Task<BulkUploadAddAndApproveDraftApprenticeshipsResult> Handle(Bulk
             {
                 BulkUploadAddAndApproveDraftApprenticeships = command.BulkUploadAddAndApproveDraftApprenticeships,
                 ProviderId = command.ProviderId,
+                LogId = command.FileUploadLogId,
```

Comment:
There is a test covering this handler 'WhenAddAndApprovingBulkUploadDraftApprenticeships' and we should ensure we cover the LogId in it. I have to admit it's definitely a really bad layout of a test but we are being remiss if we don't cover this new behaviour

Reply:
Thanks Paul, was good to add, but i've since deleted the whole lot. If we need I can create again.